packages/app/obojobo-document-engine/__tests__/Viewer/util/api-util.test.js

```diff
@@ -154,6 +157,9 @@ describe('apiutil', () => {
 		}).then(() => {
 			expect(fetch).toHaveBeenCalled()
 			expect(window.parent.postMessage).not.toHaveBeenCalled()
+			expect(spy).toHaveBeenCalledWith('mockValue')
+
+			spy.mockRestore()
```

Comment:
mock console is added in #866 (not merged yet) that we can use to update these console spies that'll be nice